src/ngraph/op/op_tbl.hpp

```diff
@@ -93,6 +93,7 @@ NGRAPH_OP(MaxPool, ngraph::op)
 NGRAPH_OP(MaxPoolBackprop, ngraph::op)
 NGRAPH_OP(Min, ngraph::op)
 NGRAPH_OP(Minimum, ngraph::op)
+NGRAPH_OP(DistBroadcast, ngraph::op)
```

Comment:
Should move this up to be in alphabetical order.

Reply:
Fixed this.